### PR TITLE
Prac 5 feedbacks

### PR DESCRIPTION
**Pull request by Shoonleiphyu**

### CODE REVIEW

File: emails.py
Comment (LinLaePhway) [inline emails.py:1]: Function user_name is so simple and easy to read. I didn't know that this step can be done in one line.

File: hex_colours.py
Comment (LinLaePhway) [inline hex_colours.py:1]: Great. I got to learn another way to write this program. In my code, there was no for loop.

File: state_names.py
Comment (LinLaePhway) [inline state_names.py:1]: Wow. Made smart changes rewriting the code.

File: wimbledon.py
Comment (LinLaePhway) [inline wimbledon.py:1]: What a clean code!!!!!

File: word_occurrences.py
Comment (LinLaePhway) [inline word_occurrences.py:1]: Your code is perfect. Got to know my code is not good enough~~~ Have to change a lot *cry *cry

**Comment by ThikeAung** [comment]
Can't find any error and found satisfactory.

**Pull request merged**